src/openms/source/DATASTRUCTURES/Adduct.cpp

```diff
@@ -72,7 +72,7 @@ namespace OpenMS
     label_(label)
   {
     if (amount < 0)
-      std::cerr << "Attention: Adduct received negative amount! (" << amount << ")\n";
+      { std::cerr << "Attention: Adduct received negative amount! (" << amount << ")\n"; }
```

Comment:
we usually do this style
```suggestion
    { 
      std::cerr << "Attention: Adduct received negative amount! (" << amount << ")\n"; 
    }
```